.github/workflows/build-and-test-Linux.yaml

```diff
@@ -6,6 +6,7 @@ on:
       - '**'
 
 jobs:
+  # This job takes approximately 18 minutes
```

Comment:
I understand that these are informational only, but how are we going to track any changes to these numbers?

Or is the plan to be more laid back and only update them if we casually observe any change?

Reply:
For now I was thinking keep this informal, but maybe we should go a little further and do actual checks? Don't know, I'm open to suggestions.